### PR TITLE
Formatting nits

### PR DESCRIPTION
**Pull request by x1ddos**
If you care. Feel free to ignore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,36 +31,37 @@ devices in the wild, so please consider sharing them for science.
...
 the Android Debug Bridge (adb) to push the binary to your device's
 /data/local/tmp/ directory. You can install adb by doing a `sudo apt-get install
 android-tools-adb` (on Ubuntu) or by installing the Android SDK via
 [android.com](https://developer.android.com/studio/index.html#downloads). Then
 do a:
-```
-make install
-make test
-```
+
```
Comment (bjorn3) [inline README.md:55]: Please keep backticks.

File: README.md [hunk 2]
```diff
@@ -31,36 +31,37 @@ devices in the wild, so please consider sharing them for science.
...
+    make test
+
 to install and start the Rowhammer test binary. Once installed, you may also
 invoke it from the shell directly:
-```
-adb shell
-cd /data/local/tmp
-./rh-test
-```
+
```
Comment (bjorn3) [inline README.md:61]: Same here.

**Review by bjorn3 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (x1ddos) [reply README.md:55]: It's easier to read without the backticks in a terminal. They are not required for simple code blocks: http://markdown-guide.readthedocs.io/en/latest/basics.html#code-block.

Github adheres to the markdown syntax, too. You can preview the result of this change here: https://github.com/x1ddos/drammer/blob/9e24acac68157b42b6b8952e8df3943afe8eef2b/README.md.

File: README.md (hunk 2 above)
Comment (x1ddos) [reply README.md:61]: See comment above.

File: README.md (hunk 1 above)
Comment (bjorn3) [reply README.md:55]: Didnt knew that you could do that.

**Comment by x1ddos** [comment]
Rebased onto master, resolving conflicts. PTAL.

**Review by m000 — CHANGES_REQUESTED** [review]
Looks nice. Could you also sync the Native installation part in README.md with the changes in the Makefile?

**Pull request merged**